source/time.tex

```diff
@@ -10481,15 +10475,26 @@
 represented by \fmtgrammarterm{literal-char}
 are copied unchanged to the output.
 
+\pnum
+A \defnadj{formatting}{locale} is an instance of \tcode{locale}
+used by a formatting function, that is determined as follows:
```

Comment:
Problem with paper: This doesn't lead into the bullets that follow (we've ended a sentence here yet the bullets that follow are incomplete by themselves).
"A /formatting locale/ FL is an [...] that is determined as follows. FL is:"
Optionally, instead of "FL is:", each bullet could start a sentence with "FL is".
Also, should the bullets begin with otherwise?  Or end with "or"? 

I suggest the following:
A /formatting locale/ FL is an instance of locale used by a formatting function F, determined as follows.
* If the L option is not present in /chrono-format-spec/, FL is the "C" locale.
* Otherwise, if a locale was passed to F, FL is that locale.
* Otherwise,  FL is the global locale.

Please consider applying my suggested change as a separate commit, else let's open an edit issue.

Reply:
I would just delete "determined as follows:". What do you think? @burblebee, @jensmaurer? (To be done as a later commit.)

Reply:
@burblebee , please create a separate editorial issue for this one.
I'll need to check more of the surroundings to form an opinion.